python/ray/ray_constants.py

```diff
@@ -178,19 +178,27 @@ def to_memory_units(memory_bytes, round_up):
 
 LOG_MONITOR_MAX_OPEN_FILES = 200
 
+# The object metadata field uses the following format: The first character
+# indicates the type of object (CROSS_LANGUAGE, PYTHON, RAW, ACTOR_HANDLE)
+# according to the variables below. After that comes an optional debug string
+# that starts with the DEBUG_PREFIX.
+
 # A constant used as object metadata to indicate the object is cross language.
-OBJECT_METADATA_TYPE_CROSS_LANGUAGE = b"XLANG"
+OBJECT_METADATA_TYPE_CROSS_LANGUAGE = b"X"
 # A constant used as object metadata to indicate the object is python specific.
-OBJECT_METADATA_TYPE_PYTHON = b"PYTHON"
+OBJECT_METADATA_TYPE_PYTHON = b"P"
 # A constant used as object metadata to indicate the object is raw bytes.
-OBJECT_METADATA_TYPE_RAW = b"RAW"
+OBJECT_METADATA_TYPE_RAW = b"R"
```

Comment:
Can we preserve the original metadata strings please? The new ones are hard to understand.

Reply:
Fixed!